src/imports.rs

```diff
@@ -29,7 +29,6 @@ impl<'a> FmtVisitor<'a> {
                             path_list: &[ast::PathListItem],
                             visibility: ast::Visibility,
                             vp_span: Span) -> String {
-        // FIXME check indentation
         let l_loc = self.codemap.lookup_char_pos(vp_span.lo);
```

Comment:
This is still using the left hand side of the item to calculate the indent (and thus the space remaining) for the import. You need to pass in the indent from the caller of this method.